pkg/agent/flowexporter/connections/connections.go

```diff
@@ -15,48 +15,54 @@
 package connections
 
 import (
+	"container/heap"
 	"fmt"
 	"sync"
+	"time"
 
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/klog/v2"
 
 	"antrea.io/antrea/pkg/agent/flowexporter"
+	"antrea.io/antrea/pkg/agent/flowexporter/exporter/priorityqueue"
 	"antrea.io/antrea/pkg/agent/interfacestore"
 	"antrea.io/antrea/pkg/agent/proxy"
 )
 
-type connectionStore struct {
-	connections   map[flowexporter.ConnectionKey]*flowexporter.Connection
-	ifaceStore    interfacestore.InterfaceStore
-	antreaProxier proxy.Proxier
-	mutex         sync.Mutex
+type ConnectionStore struct {
+	Connections         map[flowexporter.ConnectionKey]*flowexporter.Connection
```

Comment:
why is this map treated as Public?

Reply:
Previously was using the map lock in exporter package, forgot to change back.. will do